src/classes/Game.js

```
@@ -39,6 +39,11 @@ class Game extends EventEmitter {
     /* There's a limit of players so there should be a check before adding
     However, this limit would be dependant on the guild.
     We still need to define how these guild settings are saved. */
+    this._validateBeforeAddingPlayer();
+
+    if (this.players.length >= 10) {
+      return;
+    }
 
```

This if check should be inside the `_validateBeforeAddingPlayer()` method because it's an actual validation to tell if the player can join.

Also, this means that if this condition is met, it should throw an error. You probably need to create a new `GamePlayersError` that extends `GameError` in `./src/classes/Erros.js` or something like that. If you do end up making a new error class it should have a similar structure as the other errors (if necessary) and should have its tests.